client/signup/steps/plans/index.jsx

```diff
@@ -86,6 +86,7 @@ module.exports = React.createClass( {
 				<PlanList
 					plans={ this.state.plans }
 					comparePlansUrl={ this.comparePlansUrl() }
+					flowName={ this.props.flowName }
```

Comment:
We can avoid adding more signup-specific names to `PlanList` and its children by changing this prop name to something like `enableFreeTrials`, and checking the flow name here while we're still in a `client/signup/` file:

```js
enableFreeTrials={ 'free-trials' === this.props.flowName }
```